mend_import_sbom/import_sbom.py

```diff
@@ -182,8 +190,24 @@ def csv_to_json(csv_file):
     return dep
 
 
+def run_api(header, data, agent=False):
```

Comment:
Also, in preparation for API 2.0 (where we have multiple endpoints with multiple methods), instead of using `requests.post(...)`, use `requests.request(method, ...)` and add `method` param to the function definition. Optionally, the `method` parameter can default to "POST" for now as we're still using API 1.4.